clientv3/README.md

```diff
@@ -72,6 +72,14 @@ if err != nil {
 }
 ```
 
+## Metrics
+
+etcd client provides metrics reporting. This can enable the operators to know how clients are performing and which client generates the most load.
```

Comment:
@heyitsanthony the developer can be a new user trying to use etcd. And a fresh developer may need to know what metrics does, in order to use it. But i don't have any strong preference. If you feel whatever i suggested is not necessarily required will do the changes exactly as you mentioned.